meters/sunspec/sunspec.go

```diff
@@ -302,7 +300,16 @@ func (d *SunSpec) Query(client modbus.Client) (res []meters.MeasurementResult, e
 			// sort blocks so block 0 is always read first
 			sortedBlocks := make([]int, len(blockMap))
 			for k := range blockMap {
-				sortedBlocks = append(sortedBlocks, k)
```

Comment:
Das führt dazu, dass nur ein Block gelesen werden kann, eben der erste. Mir ist nicht richtig klar, welchem Zweck das dient. Kannst Du denn damit pbehaupt irgendwas aus dem zweiten Device auslesen? Meine Vermutung wäre eher nein?

Reply:
Ja ich lese die Werte aus beiden devices aus, da über alle models in https://github.com/volkszaehler/mbmd/blob/master/meters/sunspec/sunspec.go#L294 iteriert wird. Ja #131 war die Basis, dass ich soweit gekommen bin. Ich hab es gerade nochmal ohne die Änderung probiert und ich bekomme das selbe Ergebnis wie in #131. In dem einzelnen Model ist nur ein Block vorhanden.